cordovasim/plugins/org.jboss.tools.cordovasim.eclipse/META-INF/MANIFEST.MF

```diff
@@ -24,17 +24,17 @@ Require-Bundle: org.jboss.tools.browsersim.browser;bundle-version="3.7.0",
  org.eclipse.wst.sse.ui;resolution:=optional,
  org.jboss.tools.common.text.ext;bundle-version="3.3.0";resolution:=optional,
  org.jboss.tools.jst.web.ui;bundle-version="3.3.0";resolution:=optional,
- org.eclipse.jetty.continuation;bundle-version="[9.2.13,9.2.14)",
- org.eclipse.jetty.http;bundle-version="[9.2.13,9.2.14)",
- org.eclipse.jetty.io;bundle-version="[9.2.13,9.2.14)",
- org.eclipse.jetty.security;bundle-version="[9.2.13,9.2.14)",
- org.eclipse.jetty.server;bundle-version="[9.2.13,9.2.14)",
- org.eclipse.jetty.servlet;bundle-version="[9.2.13,9.2.14)",
- org.eclipse.jetty.servlets;bundle-version="[9.2.13,9.2.14)",
- org.eclipse.jetty.util;bundle-version="[9.2.13,9.2.14)",
- org.eclipse.jetty.proxy;bundle-version="[9.2.13,9.2.14)",
- org.eclipse.jetty.client;bundle-version="[9.2.13,9.2.14)",
- org.eclipse.jetty.rewrite;bundle-version="[9.2.13,9.2.14)",
+ org.eclipse.jetty.continuation;bundle-version="[9.0.0,10.0.0)",
```

Comment:
Add this wide dependency range for test plugins only if you need it for testing purposes.